### PR TITLE
Fixed issues of 2352 and 2245

### PR DESCRIPTION
**Pull request by shabrikaruna**
**Description (required)**

Fixes #{GitHub issue number} {Github issue title}

What changes did you make and why?

**Tests performed (required)**

Tested {build variant, e.g. ProdDebug} on {name of device or emulator} with API level {API level}.

**Screenshots showing what changed (optional - for UI changes)**

Need help? See https://support.google.com/android/answer/9075928

---

_Note: Please ensure that you have read CONTRIBUTING.md if this is your first pull request._

### CODE REVIEW

File: app/src/main/res/layout/bottom_sheet_nearby.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
     </android.support.constraint.ConstraintLayout>
 
     <FrameLayout
-        android:id="@+id/container_sheet"
+            android:id="@+id/container_sheet"
```
Comment (nicolas-raoul) [inline app/src/main/res/layout/bottom_sheet_nearby.xml]: Could you please submit a new pull request without indentation/spacing changes? If you want to submit indentation/spacing changes it is OK but please do it in another pull request. Solving only one issue per pull request makes review much easier.
Thanks a lot! :-)

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -17,25 +17,26 @@
     <uses-permission android:name="android.permission.MANAGE_DOCUMENTS" />
     <uses-permission android:name="com.google.android.apps.photos.permission.GOOGLE_PHOTOS" />
     <uses-permission android:name="android.permission.READ_LOGS" />
-    <uses-permission android:name="android.permission.SET_WALLPAPER"/>
+    <uses-permission android:name="android.permission.SET_WALLPAPER" />
```
Comment (nicolas-raoul) [inline app/src/main/AndroidManifest.xml]: This too.

File: app/src/main/res/layout/bottom_sheet_nearby.xml (hunk 1 above)
Comment (shabrikaruna) [reply app/src/main/res/layout/bottom_sheet_nearby.xml]: Hey, @nicolas-raoul I am not able to create a new pull request!

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -17,25 +17,26 @@
...
         android:label="@string/app_name"
-        android:theme="@style/LightAppTheme"
         android:largeHeap="true"
-        android:hardwareAccelerated="false"
-        android:supportsRtl="true" >
-        <activity android:name="org.acra.CrashReportDialog"
-            android:theme="@android:style/Theme.Dialog"
-            android:launchMode="singleInstance"
+        android:supportsRtl="true"
+        android:theme="@style/LightAppTheme">
```
Comment (nicolas-raoul) [inline app/src/main/AndroidManifest.xml]: If I understand correctly you just reordered things? Please restart your pull request from zero and only change things that are vital for the issue's resolution. Also, please split resolution of 2352 and resolution of 2245 into two separate pull requests, thanks! :-)

**Comment by domdomegg** [comment]
Closing in favour of #2454 and #2301
